docs/state-manager.md

```
@@ -94,6 +101,11 @@ Provides:
 none → quarantined → draining → drain-succeeded → remediating → remediation-succeeded → none
 ```
 
+**No pods to drain:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)